### ISSUE TITLE
Accessibility regression in Mumble skin

### ISSUE THREAD

**Issue by fwaggle** [mumble-voip/mumble#1787]
Copy+pasted from IRC per @mkrautz 

(treaki) hi
(treaki) we have got an complaint about the skin of the new mumble version
(treaki) the new skin isnt barrier-free, the chat window isnt usable for people with handicapped accessible...
(treaki) my handicapped communication partner is using mumble version 1.3.0 704g6fe15547 (verbal communicated) with windows vista
(treaki) the problem is that she isnt able anymore to scroll the chat window because of the change of the design
(treaki) it lookes like this on her system:
(treaki) http://treaki.tk/upload/uploads/1439803942/150817_rolli_problem_mumble.jpeg
(treaki) she would like to have a design that is looking like this:
(treaki) http://treaki.tk/upload/uploads/1439804082/150817_rolli_problem_mumble02.jpeg
(treaki) she also have tried out with someone else to change the theme files by coping it into the installation folder as it was possible in earlier mumble versions
(treaki) that also didnt worked ...
(treaki) she told me that it lokes for here like 'willed and failed', or disimprove
(treaki) for me it lookes like if the developers have tryed to act as windows 10 on an windows vista system with enabled old high contrast design
(treaki) please fix that in the next mumble version!
(treaki) also the options which were present in changing the style on old mumble versions are missing and there are only 3 not suitable options left
(treaki) (here a screenshot of the option which i mean from my system:)
(treaki) http://treaki.tk/upload/uploads/1439803271/Screenshot%20from%202015-08-17%2011:20:36.png
(treaki) also she told me right now that she was in earlier versions able to use dragon natrual speaking with windows visata and 7 and mumble
(treaki) but with that new version of mumble it is also not possible anymore
(treaki) what have you done there?
(treaki) lookes like you have imported the windows 10 toolkit into older versions
(treaki) we would like to have a possiblity to use the features of the new mumble versions with the classic designs and the classic toolkits
(treaki) so it is possible to access them in a barriar-free way!!

**Comment by mkrautz** [comment]
Thanks @fwaggle!

**Comment by mkrautz** [comment]
Comments were made by "treaki" on #mumble/freenode.

**Comment by hacst** [comment]
I guess the first thing I get from this is: "Really shouldn't forget to add that browse-to-theme-folder-link" I wanted to add before release. I wonder whether themeing actually affects any of the accessibility features directly. Dragon not working anymore would be more of a Qt thing we might want to look into.

**Comment by Krzmbrzl** [comment]
This issue seems not relevant anymore as we can probably assume that all Windows users are using Win10 anyway. So using Win10-style themes seems fine?
